app/components/search.tsx

```diff
@@ -109,17 +109,22 @@ export default function Search({
                   }}
                 />
               ))}
+              <a
+                href={`https://stampy.ai/wiki/Special:FormStart?form=Q&page_name=${searchInput}`}
+                target="_blank"
+                rel="noreferrer"
+                className="result-item none-of-the-above row m-spread x-center"
```

Comment:
I'm happy if we decided to switch to utility classes, but it would ideally be https://tailwindcss.com/ or some other library that would be known to other developers instead of rolling our own...

and we could even slowly transition from our current system ("semantic" classes I guess) to utility classes by converting only 1 component at a time and not the whole app...

but I am less happy about mixing semantic and utility classes inside the same element withing one `className`

if you promise you will be working on converting the whole file to utility styles, removing the old classes, ifthis is just temporary mixing of classes from 2 CSS files without explicit imports, then feel free to merge

but we could get away with plain CSS without CSS Modules or other more sophisticated solutions only because we kept all CSS in 1 file that is easy to check for name collisions and unused class names,, adding more plain-CSS files that are imported only in root is asking for trouble IMHO 🤔

Reply:
Heh, looks like I didn't scroll enough before sending my other reply. In my own experience I've found that one large sheet doing everything becomes tricky in really silly / mundane ways, it just gets hard to scroll around etc. If we'd rather just use Tailwind I'd be ok with that and will get to work on the relevant changes now.